SearchApplication/Controllers/ReportingController.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using MySql.Data;
+using DCode.Data.DbContexts;
+using SearchApplication.Models;
+using MySql.Data.MySqlClient;
+using System.IO;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace SearchApplication.Controllers
+{
+    public class ReportingController : Controller
+    {
+        
+       
+        // GET: Reporting
+        public ActionResult TopRequestors()
+        {
+            string conString = System.Configuration.ConfigurationManager.ConnectionStrings["DcodeConnectionString"].ToString();
```

Comment:
A couple of observations here:

1) Can we use `using` blocks to auto-dispose of connection and command objects. 
2) Also, pulling up EDM objects directly into the controller layer isn't a good idea.
3) Would be best if we closed connections in a `finally` block.
4) Can store the connection string as a constant at the controller level.